guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -53,12 +69,12 @@ To fork the devdocs repository on Github, do the following:
 
 1.  Clone the repository into your development environment and then start writing and committing your changes. Optionally, create a branch first if you plan to work on multiple changes.
 
-1.  Build the devdocs site locally using Jekyll by following the instructions in our [README](https://github.com/magento/devdocs/blob/develop/README.md){:target="\_blank"}.
+1.  Build the devdocs site locally using Jekyll by following the instructions in our [README](https://github.com/magento/devdocs/blob/develop/README.md){:target="_blank"}.
 
 ## Update your fork {#sync}
 As we merge changes with the upstream repository, your fork becomes outdated and pull requests might result in merge conflicts. To see if your fork is outdated, open the fork page in GitHub. If you see the following message at the top of the page, you must update your fork: `This branch is <number> commits behind develop`.
 
-There are two ways to update your fork. The typical way is discussed in [GitHub documentation](https://help.github.com/articles/syncing-a-fork){:target="\_blank"}. Make sure to update from the correct branch!
+There are two ways to update your fork. The typical way is discussed in [GitHub documentation](https://help.github.com/articles/syncing-a-fork){:target="_blank"}. Make sure to update from the correct branch!
```

Comment:
Maybe "Ensure you are updating your fork from the correct branch."

Reply:
"Navigate to the GitHub page for your forked repository and click New pull request".

Reply:
edited